src/pages/test-report/components/end-test-modal/__tests__/end-test-modal.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { EndTestModal } from '../end-test-modal';
+import { IonicModule, NavParams } from 'ionic-angular';
+import { NavParamsMock } from 'ionic-mocks';
+import { AppModule } from '../../../../../app/app.module';
+
+describe('EndTestModal', () => {
+  let fixture: ComponentFixture<EndTestModal>;
+  let component: EndTestModal;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        EndTestModal,
+      ],
+      imports: [
+        AppModule,
+        IonicModule,
+      ],
+      providers: [
+        { provide: NavParams, useFactory: () => NavParamsMock.instance() },
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(EndTestModal);
+        component = fixture.componentInstance;
+      });
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+  });
+
+  describe('DOM', () => {
```

Comment:
No DOM tests?

Reply:
I've added a couple of tests now for the buttons